PHP/Persistence/README.md

```diff
@@ -0,0 +1,11 @@
+# Persistence:
+
+If you were to multply the digits of an integer multiple times, you would discover the sequence of products will always end with a single digit number. This single digit number is also known as a multiplicative persistence. Few examples:
+
+814 -> 24 -> 8
```

Comment:
You need to give a better description here. 

8 * 1 * 4 = 24 -> 2*4 = 8 .  this took two steps so the multiplicative persistence of 814 is 2